dropshot/examples/https.rs

```diff
@@ -0,0 +1,151 @@
+// Copyright 2020 Oxide Computer Company
+/*!
+ * Example use of Dropshot.
```

Comment:
I think this is just copied/pasted.  Can we give a better description of this example?